shared/src/libs/types/get-items-by-query-dto.type.ts

```diff
@@ -0,0 +1,5 @@
+type GetItemsByQueryDto = {
```

Comment:
```suggestion
type JournalEntriesFilteringDto = {
```
and move to appropriate package

Reply:
@Liza-Veis @v0ldemar01 told me to make one schema for both journal and chats in this [comment](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/339#discussion_r1335015587) so i make the same for query type

Reply:
I think we may have different queries for entities/packages, so I would prefer to split them by entities

Reply:
Let's rename it to EntiesyFilteringDto and the same style for validation schema

Reply:
done